flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkScan.java

```diff
@@ -83,8 +74,13 @@
           .bucket("id", 1)
```

Comment:
FLIP-27 source will have a test extending from this base class. Logic for the current source is refactored into the `TestFlinkSource` class